resources/aws/routetable.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+type RouteTable struct {
+	Name  string
+	VpcID string
+	id    string
```

Comment:
If it's an exported field then it should be ID, but in this case I think @nhlfr wanted to make this field private.